railties/lib/rails/tasks/statistics.rake

```diff
@@ -1,7 +1,18 @@
 # While global constants are bad, many 3rd party tools depend on this one (e.g
 # rspec-rails & cucumber-rails). So a deprecation warning is needed if we want
 # to remove it.
```

Comment:
Technically `TEST_TYPES` is nested under `CodeStatistics` which is a `nodoc` class... so we explicitly say that this is internal. We're also just trading one constant for another, with the new one being more useful :)